slides/props.md

```diff
@@ -0,0 +1,46 @@
+### Props
+
+
+---
+
+#### Props
+
+- An object passed through the parent, received as an argument
+- Added as *attributes* on the component
+
+
+```
```

Comment:
Need to specify JavaScript as the language to enable highlighting. 

```
```js
ReactDOM.render(
```